src/components/DeckBuilderDryRunView/index.js

```diff
@@ -2,18 +2,53 @@ import React from 'react'
 import modifyDeck from '../../helpers/modifyDeck'
 import DryRunner from '../DryRunner'
 import DeckMechanisms from '../DeckMechanisms'
+import { NotificationContext } from '../NotificationProvider'
 import WikiLink from '../WikiLink'
+import { BRAWLS } from '../../constants/brawl'
 import isCard from '../../helpers/isCard'
+import isSuggestedDeck from '../../helpers/isSuggestedDeck'
+
+const getPresetOptions = (suggestedDeck, sendNotification) => {
+  const presetOptions = {
+    modifier: 'NONE',
+    equals: false,
+  }
+  if (suggestedDeck) {
+    if (suggestedDeck.category === 'BRAWL') {
+      presetOptions.modifier = suggestedDeck.brawl
+      const brawlLabel = BRAWLS.find(brawl => brawl.id === suggestedDeck.brawl)
+        .label
+      sendNotification(`Brawl deck found. Loaded with modifier ${brawlLabel}.`)
+    }
+    if (suggestedDeck.category === 'EQUALS') {
+      presetOptions.equals = true
+      sendNotification('Tournament deck found. Loaded in equals mode.')
+    }
+  }
+  return presetOptions
+}
 
 export default props => {
   const params = new URLSearchParams(window.location.search)
   // The mode is theoretically not quite supposed to be changed at run time, but
   // this is a workaround to be able to pick an initial hand for testing
   // purposes. The mode is restored to `AUTOMATIC` as soon as the 4th card has
   // been picked.
+  const { notify } = React.useContext(NotificationContext)
+  const sendNotification = React.useCallback(
+    message => notify({ icon: 'sword', children: message }),
+    [notify]
+  )
   const [mode, setMode] = React.useState(params.get('mode') || 'AUTOMATIC')
   const [modifier, setModifier] = React.useState('NONE')
   const [equalsMode, setEqualsMode] = React.useState(false)
+  // If the deck is saved as brawl/tournament, load the dry-runner in the correct mode
+  React.useEffect(() => {
+    const suggestedDeck = isSuggestedDeck(props.deck)
+    const presetOptions = getPresetOptions(suggestedDeck, sendNotification)
```

Comment:
I think `getPresetOptions` should receive `props.deck` (instead of a suggested deck), check if it’s a suggested deck itself, and return the appropriate presets.